realis.go

```diff
@@ -320,6 +322,28 @@ func (r realisClient) AddInstances(instKey aurora.InstanceKey, count int32) (*au
 	return response.ResponseCodeCheck(resp)
 }
 
+//Scale down the number of instances under a job configuration using the configuratipn of a specific instance
+func (r realisClient) RemoveInstances(key *aurora.JobKey, count int32) (*aurora.Response, error) {
+	instanceIds, err := r.GetInstanceIds(key, aurora.ACTIVE_STATES)
+	if err != nil {
+		return nil, errors.Wrap(err, "RemoveInstances: Could not retrieve relevant instance IDs")
+	}
+	if len(instanceIds) < int(count) {
+		return nil, errors.New(fmt.Sprintf("RemoveInstances: No sufficient instances to Kill - " +
+			"Instances to kill %d Total Instances %d", count, len(instanceIds)))
+	}
+	instanceList := make([]int32, count)
+	i := 0
+	for k := range instanceIds {
```

Comment:
Could we do something like `instanceList = instanceIds[:count-1]` instead of this loop?